cmd/prometheus/main.go

```diff
@@ -26,6 +29,7 @@ import (
 	"os"
 	"os/signal"
 	"path/filepath"
+	"plugin"
```

Comment:
`package plugin` was always experimental and is now abandoned. It shouldn't be used.